components/brave_new_tab_ui/components/default/braveToday/customize/FeedCard.tsx

```diff
@@ -132,7 +81,7 @@ export function DirectFeedCard (props: {
 }) {
   const [loading, setLoading] = useState(false)
   return <Flex direction="column" gap={8}>
-    <Card backgroundColor={getCardColor(props.feedUrl)}>
+    <Card backgroundColor={getCardColor(props.feedUrl)} data-feed-card-is-followed={true}>
```

Comment:
Do we need to do this here? I think if the direct feed is followed we will hide it and just show a channel card instead

Reply:
Only because it's using the same styles, hence the `true` instead of a variable